quantum/pointing_device/pointing_device.h

```diff
@@ -95,11 +95,21 @@ typedef enum {
 #ifdef MOUSE_EXTENDED_REPORT
 #    define XY_REPORT_MIN INT16_MIN
 #    define XY_REPORT_MAX INT16_MAX
-typedef int32_t clamp_range_t;
+typedef int32_t xy_clamp_range_t;
```

Comment:
Has been included in [a separate PR](https://github.com/qmk/qmk_firmware/pull/24422).